client/src/partials/accounts/accounts.js

```diff
@@ -3,7 +3,7 @@ angular.module('bhima.controllers')
 
 AccountsController.$inject = [
   '$rootScope', 'AccountGridService', 'NotifyService', 'bhConstants',
-  'LanguageService'
+  'LanguageService', '$state'
```

Comment:
I think this is left over from our debugging session.  Can you remove it?  And the `console.log()` below.